ansible.cfg

```diff
@@ -3,6 +3,7 @@ host_key_checking = False
 retry_files_enabled = False
 deprecation_warnings = False
 roles_path = ./roles
+stdout_callback = skippy
```

Comment:
Does this require any other setup on the client's end, or it's native? Nice find, btw. Mostly just wondering.

Reply:
Native! No setup required, as that callback plugin is shipped with standard Ansible.